src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -184,4 +183,106 @@ public void Chat_007_banUser() {
     chatUserStaff.banUser(userToBeBanned);
     chatUserStaff.unBanUser(userToBeBanned);
   }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_008", "Chat", "Modals", "ChatTests"})
+  public void userBlockedCanNotCreatePrivateMessage() {
+    ChatPageObject chatUserOne = openChatForUser(userOne, userOnePassword);
+
+    switchToWindow(1);
+    SpecialVersionPage chatWindow = new SpecialVersionPage().open();
+    ChatPageObject chatUserFive = openChatForUser(userFive, userFivePassword);
+
+    switchToWindow(0);
+    chatUserOne.clickOnDifferentUser(userFive);
+    chatUserOne.selectPrivateMessageToUser(userFive);
+    chatUserOne.blockPrivateMessageFromUser(userFive);
+
+    switchToWindow(1);
+    chatWindow.refreshPage();
+    chatUserFive.clickOnDifferentUser(userOne);
+    chatUserFive.verifyBlockedUserIsUnableWritePrivateMessage(userOne);
+  }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_009", "Chat", "Modals", "ChatTests"})
+  public void userCanOpenMessageWall() {
```

Comment:
name it: 
regularUserCanOpenMessageWall
so that we know it's not chat moderator / staff